src/test/resources/no/nav/familie/ef/sak/vedtak_fra_gitt_dato/vedtak_barnetilsyn_fra_dato.feature

```diff
@@ -105,10 +105,10 @@ Egenskap: hentVedtakForBarnetilsynFraDato
 
   Scenario: Barnetilsyn med ulike barn historikk
     Gitt følgende vedtak for barnetilsyn
-      | BehandlingId | Vedtaksresultat | Fra og med dato | Til og med dato | Barn    | Utgifter |
-      | 1            | INNVILGE        | 01.2021         | 03.2021         | Id1     | 200      |
-      | 2            | INNVILGE        | 02.2021         | 03.2021         | Id1,Id2 | 100      |
-      | 2            | INNVILGE        | 04.2021         | 04.2021         | Id2     | 50       |
+      | Periodetype BT | Aktivitetstype BT | BehandlingId | Vedtaksresultat | Fra og med dato | Til og med dato | Barn    | Utgifter |
+      | ordinær        | i_arbeid          | 1            | INNVILGE        | 01.2021         | 03.2021         | Id1     | 200      |
+      | ordinær        | i_arbeid          | 2            | INNVILGE        | 02.2021         | 03.2021         | Id1,Id2 | 100      |
+      | ordinær        | i_arbeid          | 2            | INNVILGE        | 04.2021         | 04.2021         | Id2     | 50       |
```

Comment:
Hadde vi kunnet plassert disse lengre til høyre? :) 
Tenker det kan være fint å ha kolonnerne sortert sånn at man får 
behandlingId, dato/vedtaksresultat først, som for meg er det jeg leser først, litt av en "row key"
Og sen litt metadata til perioder/behandlingen, og så eventuellt kolonnene for endret og sånt?

Reply:
Fixd: [f1c58c0](https://github.com/navikt/familie-ef-sak/pull/2021/commits/f1c58c0e8b5f06c10192d1cef1972571958a99c0) 👍